platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -47,13 +47,14 @@
 
 import org.codice.ddf.platform.util.RandomNumberGenerator;
 import org.eclipse.jetty.server.session.AbstractSession;
-import org.eclipse.jetty.server.session.AbstractSessionIdManager;
 
 /**
  * Custom implementation of the {@link org.eclipse.jetty.server.SessionIdManager} that shares session
  * data between sessions in a cluster.
  */
-public class HashSessionIdManager extends AbstractSessionIdManager {
+@edu.umd.cs.findbugs.annotations.SuppressWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "not changing")
+public class HashSessionIdManager extends org.eclipse.jetty.server.session.HashSessionIdManager {
+    //changed this to extend Jetty's impl instead of the abstract class because of a hack in pax web
```

Comment:
Pax Web is expecting an instance of the Jetty implementation, which ours was not. That ended up skipping a bunch of code that we actually need. There are also additional changes to our id manager that aren't in the Jetty Impl. This is what we use to hook in SSO.